src/main.py

```diff
@@ -39,16 +39,33 @@ def main() -> None:
         log.warning("Stopping CoinTaxman.")
         return
 
+    # Merge identical operations together, which makes it easier to match
+    # buy/sell to get prices from csv, match fees and reduces database access
+    # (as long as there are only one buy/sell pair per time,
+    # might be problematic otherwhise).
```

Comment:
Noticed a small spelling mistake here.
```suggestion
    # might be problematic otherwise).
```